src/components/toolbars/toolbar/toolbar/toolbar.tsx

```diff
@@ -283,17 +285,14 @@ export class Toolbar implements ComponentInterface {
     const range: Range | undefined = selection?.getRangeAt(0);
     const rect: DOMRect | undefined = range?.getBoundingClientRect();
 
-    const containerRect: DOMRect | undefined = this.containerRef?.getBoundingClientRect();
+    //const containerRect: DOMRect | undefined = this.containerRef?.getBoundingClientRect();
+    const styloContainerRect = (this.tools.parentNode as ShadowRoot).host.getBoundingClientRect();
```

Comment:
isn't `this.el.shadowRoot` the same as `this.tools.parentNode`?

if you need the host `this.el`